parser/parser.go

```diff
@@ -410,25 +411,31 @@ func (p *Parser) parseFieldType(pkg *packages.Package, obj types.Object) (FieldT
 	if ftype.IsObject {
 		ftype.JSType = "object"
 		ftype.SwiftType = "Any"
+		ftype.PythonType = "Any"
 	} else {
 		switch ftype.TypeName {
 		case "interface{}":
 			ftype.JSType = "any"
 			ftype.SwiftType = "Any"
+			ftype.PythonType = "Any"
 		case "map[string]interface{}":
 			ftype.JSType = "object"
 			ftype.SwiftType = "Any"
+			ftype.PythonType = "Any"
 		case "string":
 			ftype.JSType = "string"
 			ftype.SwiftType = "String"
+			ftype.PythonType = "str"
 		case "bool":
 			ftype.JSType = "boolean"
 			ftype.SwiftType = "Bool"
+			ftype.PythonType = "bool"
 		case "int", "int16", "int32", "int64",
 			"uint", "uint16", "uint32", "uint64",
 			"float32", "float64":
 			ftype.JSType = "number"
 			ftype.SwiftType = "Double"
+			ftype.PythonType = "int"
```

Comment:
will this type also handle floating point numbers?

Reply:
Good catch. I believe that should be a float. Will update.